hnn_core/tests/test_network.py

```diff
@@ -82,6 +82,48 @@ def test_network_models():
         assert np.all(np.diff(k_gbar, n=2) > 0)  # positive 2nd derivative
 
 
+def test_network_cell_positions():
+    """"Test manipulation of cell positions in the network object"""
+
+    net = jones_2009_model()
+    assert np.isclose(net.inplane_distance, 1.)
+    assert np.isclose(net.layer_separation, 1307.4)
+    pos_dict = deepcopy(net.pos_dict)
+    net._reset_cell_positions()
+    # assert reset does nothing
+    for cell_name, coords in pos_dict.items():
+        if cell_name == 'origin':
+            continue
+        for xyz_orig, xyz_new in zip(coords, net.pos_dict[cell_name]):
+            for orig, new in zip(xyz_orig, xyz_new):
+                assert np.isclose(orig, new)
+
+    net.inplane_distance = 2.
+    # check that in-plane distance has doubled
+    assert np.isclose(2. * pos_dict['L5_pyramidal'][1][1],
```

Comment:
but this is not inplane distance. It's the absolute position, no?

Reply:
hmm, not sure I understand. I guess it's more accurately a 'step size'?